indico/modules/events/abstracts/settings.py

```diff
@@ -53,6 +71,14 @@ class BOACorrespondingAuthorType(RichEnum):
 }
 
 
+BOALinkFormatting.__titles__ = {
+    BOALinkFormatting.frame: _('Frame'),
+    BOALinkFormatting.colorlinks: _('Color Links'),
+    BOALinkFormatting.ocgcolorlinks: _('Color Links for screen only'),
+    BOALinkFormatting.ocgx2colorlinks: _('Color Links for screen only(with ogcx2)'),
+    BOALinkFormatting.hidelinks: _('Hide Links')
```

Comment:
"Do not highlight links" or something like that would be better. I actually had to google to find out how the links are "hidden" (the answer is; "by using the normal text color/format")...